src/routes/retro-certs.js

```diff
@@ -9,6 +9,37 @@ const weeksCompleted = require("../utils/checkFormData");
 function createRouter() {
   const router = Router();
 
+  async function staffViewAuthStatus(postJson, responseJson) {
+    responseJson.status = AUTH_STRINGS.statusCode.userNotFound;
+    const userRecord = await cosmos.getUserByNameDobSsn(
+      postJson.lastName || "",
+      postJson.dob,
+      postJson.ssn
+    );
+    if (!userRecord) {
+      console.log("failed staff view login");
+      return responseJson;
+    }
+
+    console.log("staff view login", userRecord.id);
+    userRecord.staffLastViewedAt = Date.now();
+    await cosmos.updateUserData(userRecord);
+
+    responseJson.status = AUTH_STRINGS.statusCode.ok;
+    responseJson.weeksToCertify = userRecord.weeksToCertify;
+    responseJson.programPlan = userRecord.programPlan;
+    responseJson.lastName = postJson.lastName;
```

Comment:
I think you have to also return a boolean for whether the user has previously logged in or not, so the new "in progress" badge can be displayed

Reply:
nevermind it's in the form data